wire-ios-mono.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,24 +1,6 @@
 {
   "object": {
     "pins": [
-      {
```

Comment:
Should this be here?

Reply:
I removed it by accident. Just fixed it. Good catch!